charts/eoapi/profiles/experimental.yaml

```diff
@@ -206,6 +206,8 @@ browser:
   enabled: true
   settings:
     resources: {}
+  # STAC Browser needs external OIDC URL (accessible from user's browser)
+  oidcDiscoveryUrl: "http://localhost/mock-oidc/.well-known/openid-configuration"
```

Comment:
General q - Is this _typically_ called the Discovery URL?

`oidcConfigurationUrl`?